### ISSUE TITLE
publish to npm registry

### ISSUE THREAD

**Issue by liuyoshio** [zcaceres/fetch-mcp#1]
this is so cool!

**Comment by zcaceres** [comment]
@liuyoshio Do you have an example of an MCP that's published to NPM in a way that you find easy to use?

**Comment by wtfzambo** [comment]
> [@liuyoshio](https://github.com/liuyoshio) Do you have an example of an MCP that's published to NPM in a way that you find easy to use?

mah, this one for instance: https://www.npmjs.com/package/@modelcontextprotocol/server-github

So you can easily run it with `npx`.

**Comment by samhvw8** [comment]
@zcaceres  can you publish it :'(

**Comment by zcaceres** [comment]
@samhvw8 it looks like @pwilkin has a version published on npm. I will work with him to get package rights and link it to this repo.

**Comment by pwilkin** [comment]
> [@samhvw8](https://github.com/samhvw8) it looks like [@pwilkin](https://github.com/pwilkin) has a version published on npm. I will work with him to get package rights and link it to this repo.

Added you as maintainer (you have to accept the invitation I think).

**Comment by zcaceres** [comment]
> > [@samhvw8](https://github.com/samhvw8) it looks like [@pwilkin](https://github.com/pwilkin) has a version published on npm. I will work with him to get package rights and link it to this repo.
> 
> Added you as maintainer (you have to accept the invitation I think).

I'm in -- thanks! Any interest in maintaining this package more generally?

**Comment by pwilkin** [comment]
> > > [@samhvw8](https://github.com/samhvw8) it looks like [@pwilkin](https://github.com/pwilkin) has a version published on npm. I will work with him to get package rights and link it to this repo.
> > 
> > 
> > Added you as maintainer (you have to accept the invitation I think).
> 
> I'm in -- thanks! Any interest in maintaining this package more generally?

Yeah, I've been thinking about a few improvements recently, didn't know if you were still active, but will gladly contribute / look over stuff. I've been building my own MCP tooling in areas where I feel there's a shortage and your fetch was one of the first places I stumbled upon when looking for existing solutions, hence the fork and changes.

**Comment by zcaceres** [comment]
Published to npm as mcp-fetch-server@1.1.0. Install with `npm install -g mcp-fetch-server`.
